iOS Client/ArticlesController/MBArticlesViewController.swift

```diff
@@ -17,7 +17,40 @@ class MBArticlesViewController: UIViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view, typically from a nib.
+        /******** START EXAMPLE ***********/
```

Comment:
this is example code so u can quickly see everything working when you spin this project up